templates/compute_pulse_heights.ipynb

```diff
@@ -2,139 +2,37 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    data_dir = os.path.join('..', 'toffy', 'data', 'tissue')
</pre><p>This should be updated to match the format of the other notebooks, and should point to a run_dir inside of the bin_files dir</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/126/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>